releng/com.b2international.snowowl.server.update/assembly/common/scripts/snowowl_delta_import.sh

```diff
@@ -0,0 +1,244 @@
+#!/usr/bin/env bash
+#
+# Copyright 2019 B2i Healthcare Pte Ltd, http://b2i.sg
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+
+#
+# Snow Owl terminology server import script
+# See usage or execute the script with the -h flag to get further information.
+
+#
+# Parameters that must be configured before execution
+#
+
+# The username to authenticate through the REST API
+SNOW_OWL_USER=""
+
+# The password for the above user
+SNOW_OWL_USER_PASSWORD=""
+
+# The source folder with the exported RF2 content
+SOURCE_FOLDER=""
+
+#
+# Global variables / constants, advanced parameters
+#
+
+# Base URL of the Snow Owl server
+SNOW_OWL_BASE_URL="http://localhost:8080"
+
+# URL for Snow Owl's REST API
+SNOW_OWL_API_URL="/snowowl/snomed-ct/v3"
+
+# Media type to use for REST requests
+MEDIA_TYPE="application/vnd.com.b2international.snowowl+json"
+
+# The url of the bugzilla repository, e.g.: http://localhost:9090/bugzilla
+BUGZILLA_URL=""
+
+#Login credentials to bugzilla
+BUGZILLA_USERNAME=""
+BUGZILLA_PASSWORD=""
+
+usage() {
+
+	cat <<EOF
+
+NAME:
+
+    Delta import script for the Snow Owl terminology server
+
+    OPTIONS:
+
+    -h
+        Show this help
+    -u
+        Define a username with privileges to the Snow Owl REST API
+    -p
+        Define the password for the above user
+    -b
+        Snow Owl base URL, defaults to 'http://localhost:8080'
+    -f
+        Source folder where the exported content can be found
+    -s
```

Comment:
This option does not exist among the available parameters. Please remove this line!